package.json

```diff
@@ -36,7 +41,7 @@
     "build": "npm run clean && rollup -c --environment BASE_CODE_MIN,STANDALONE,STANDALONE_MIN && echo \"Base Code:\" && cat distTest/baseCode.min.js",
     "build:cli": "node scripts/build-alloy.js",
     "prepare": "husky && cd sandbox && npm install",
-    "prepublishOnly": "rimraf libEs5 libEs6 && babel src -d libEs5 --env-name npmEs5 && babel src -d libEs6 --env-name npmEs6",
+    "prepack": "rimraf libEs5 libEs6 && babel src -d libEs5 --env-name npmEs5 && babel src -d libEs6 --env-name npmEs6",
```

Comment:
Changed this to `prepack` in order to get a proper NPM package when I run locally the `npm pack` command.